### PR TITLE
Add default messages page

### PR DESCRIPTION
**Pull request by ishikawa999**
Add the ability to check config/locales/*.yml from admin view.
I would like a review.

ScreenShot-2019-06-11-16 17 12

Clicking the "Default messages" link will open a window.
ScreenShot-2019-06-11-16 17 33

### CODE REVIEW

**Comment by yui-har** [comment]
The Dialog opened with window.open can be resized, so if you widen it you can use the regular menu.
How about creating a dialog using jQuery's Dialog?

**Comment by ishikawa999** [comment]
@yui-har 
Thank you for the review.
> The Dialog opened with window.open can be resized, so if you widen it you can use the regular menu.

I think that changing the size of the window is not a problem.
However, as you said, using the regular menu can be confusing for the user.
How about hiding unnecessary menu bars(e.g. sidebar, header, menu_bar) by CSS?

ScreenShot-2019-06-13-9 46 44
 

> How about creating a dialog using jQuery's Dialog?

Since the jQuery's Dialog is displayed in the window, I think that the dialog gets in the way when manipulating the input form while looking at default_messages.

File: assets/stylesheets/default_messages.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+table.filecontent tr:target th.line-num {
+  background-color: #E0E0E0;
+  color: #777;
+}
+
+table.filecontent tr:target td.line-code {
+  background-color: #DDEEFF;
+}
```
Comment (ishikawa999) [inline assets/stylesheets/default_messages.css]: Hide the menu bar style

e.g.
```suggestion
}

/* Hide unnecessary headers etc */
body.action-default_messages #sidebar,
body.action-default_messages #header,
body.action-default_messages #top-menu {
  display: none;
}

body.action-default_messages #main {
  padding: 0px;
}
```

Comment (yui-har) [reply assets/stylesheets/default_messages.css]: Do you need the following CSS.
```
body.action-default_messages #main {
 padding: 0px;
}
```

**Comment by yui-har** [comment]
> How about hiding unnecessary menu bars(e.g. sidebar, header, menu_bar) by CSS?

I think it's good.

> Since the jQuery's Dialog is displayed in the window, I think that the dialog gets in the way when manipulating the input form while looking at default_messages.

You're right.

---

I am worried that the window name of window.open is Japanese.

File: assets/stylesheets/default_messages.css (hunk 1 above)
Comment (ishikawa999) [reply assets/stylesheets/default_messages.css]: When the screen is responsive, the following changes have been made to increase the height of the header.

```css
@media screen and (max-width: 899px)
#main {
    padding: 64px 0 0;
}
```
ScreenShot-2019-06-13-11 08 34

The "body.action-default_messages #main" style is to eliminate extra space.

Comment (yui-har) [reply assets/stylesheets/default_messages.css]: > When the screen is responsive, the following changes have been made to increase the height of the header.

It confirmed. I think it's good.

**Comment by ishikawa999** [comment]
> I am worried that the window name of window.open is Japanese.

It fixed in e2ecac5.

**Comment by ishikawa999** [comment]
> Since window name is simple, how about including a plug-in name in window name?

I added plugin name to the window name.
"redmine_message_customize_plugin-default_messages"

**Comment by ishikawa999** [comment]
#6

**Pull request merged**